Constructor-io/constructorio-ui-autocomplete
# update storybook addon versions

🤞🏻 hoping this plays nice with github pages 🤞🏻 

to test locally, 
1. simulate what we do in github with github actions: `npm run build-storybook`
2. then serve the new static build locally, like we do with github pages with: `npm run serve-built-storybook`

instead of this:
<img width="1438" alt="Screenshot 2022-12-28 at 3 48 26 PM" src="https://user-images.githubusercontent.com/20070114/209830320-d67d8e32-3dae-4887-9318-fb48b9082f2a.png">


I now see this:
<img width="1279" alt="Screenshot 2022-12-28 at 3 52 58 PM" src="https://user-images.githubusercontent.com/20070114/209830338-a78a0b07-5aff-4eee-9681-519d850d5a56.png">
